CHANGELOG.md

```diff
@@ -18,6 +18,13 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 More details about these changes can be found on our GitHub repo.
 
+
+## 0.39.0 - 2019-09-05
+
+### Added
```

Comment:
These entries should be moved up to the master section to be included in the next release changelog.